Jenkinsfile

```diff
@@ -9,9 +9,25 @@ pipeline {
             }
         }
 
+       stage("Yosys synthesis"){
+            steps {
+               sh 'docker run --rm -v $(pwd):/src -w /src librecores/librecores-ci:0.4.0 /bin/bash -c \
+               "fusesoc library add mor1kx /src; \
+                fusesoc run --target=synth mor1kx; \
+                /test-scripts/extract-yosys-stats.py < build/mor1kx_*/synth-icestorm/yosys.log"'
+            }
+        }
+
+        stage('Resource Usage Report Generation') {
+            steps {
+                echo "-=- execute performance tests -=-"
```

Comment:
You are still saying performance tests here.

Reply:
I am sorry @stffrdhrn  . But I will be changing this . I am working on jenkins shared library . Will send changes soon in PR with library for both or1k-marochinno and mor1kx